### ISSUE TITLE
[FEATURE] Leitweg-ID bitte um Lieferanten-Nummer ergänzen

### ISSUE THREAD

**Issue by librelegal** [jlawyerorg/j-lawyer-org#2639]
Leitweg-ID eintragen, es fehlt aber ein Feld für die [Lieferanten-Nummer.](https://www.zrb.bund.de/SharedDocs/Downloads/DE/Themen/IT-Verfahren/eRechnung/Anlage3-Lieferanteninfo.pdf?__blob=publicationFile&v=2). Diese wird vermutlich benötigt.

**Comment by j-dimension** [comment]
@librelegal die Lieferantennummer beschreibt nicht den Rechnungsempfänger, sondern den Absender - korrekt? Dann ümsste die Nummer in die Nutzerverwaltung, nicht an die Adressbucheinträge.

**Comment by librelegal** [comment]
Ja, diese wurde mit von der öffentlichen Einrichtung mitgeteilt, ich 
muss die in der Rechnung angeben

Jens Kutschke wrote on 18.11.2024 at 00:01:
> @librelegal die Lieferantennummer beschreibt nicht den Rechnungsempfänger, sondern den Absender - korrekt? Dann ümsste die Nummer in die Nutzerverwaltung, nicht an die Adressbucheinträge.
>
